.github/workflows/backport.yml

```diff
@@ -22,7 +22,9 @@ jobs:
           fetch-depth: 0
 
       - name: Set git user
-        run: .github/scripts/set-git-user.sh
```

Comment:
Just wondering, why remove this script? Seems like it was good for keeping the bot definition in one place

Reply:
I had originally extracted the script so that the workflows would be similar across different language repos using different bots, which isn't needed anymore, but ya, it's probably worth keeping the abstraction due to the weird email